bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -79,6 +79,16 @@ spec:
                           Job Service container image to be used instead of the provided
                           one by SonataFlow
                         type: string
+                      broker:
```

Comment:
do we want to match the SFP structure instead, i.e: 
```
eventing:
  description: Eventing related properties to set in the SonataFlow Platform
  properties:
    broker:
      description: Broker contains the name and namespace of the broker to use if using Knative eventing for components communications 
      properties:
        name:
          description: name describes the name of the broker to use
          type: string
        namespace:
          description: namespace describes the namespace on which the broker to use is deployed
          type: string
      type: object
type: object
```

Reply:
have you considered adding these fields to the `sonataflowOperator` structure? I'm aware that as it is here keeps consistency with the other fields (dataindex and job service image resource limits), but I find it strange that these fields reside under the `orchestrator` struct, even though they are specific to `sonataflow`. If they belong to both `orchestrator` and `sonataflow`, what do you think of the broker being its own high level entity, sibling to the orchestrator?

Reply:
@jenniferubah @masayag WDYT?

Reply:
For consistency sake, I think we can leave it at the same level with dataindex and job service images...(children of `sonataflowPlatform`) under the orchestrator.

Perhaps, for the go-operator, we split the `sonataflowOperator` (now `serverlessLogicOperator`) as we did for `rhdh`, and have the `sonataflowPlatform` configurations (dataindex, job service, broker etc) as a child since these are all specific to `sonataflow`. E.g

```
"serverlessLogic": {
  "installOperator": true
  "resources":
      "limits":
        "cpu": "cpu"
        "memory": "memory"
      "requests":
        "cpu": "cpu"
        "memory": "memory"
  "eventing":
    "broker":
      "name": "name"
      "namespace": "namespace"
}
```

Reply:
Yes please, that's sounds fine to me.

Reply:
done